### PR TITLE
round actually ends on narsie & time defines

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

better than #11030

times tweaked so the cinematic has just enough time to play before the game closes at 1m30s (90s)

round ends (not real code just approx)
`sleep(5 SECONDS)` 5s
An acausal dimensional event has been detecte...
`sleep(30 SECONDS)` 30s
Simulations on acausal dimensional event complete. Deploying ...
`sleep(5 SECONDS)` 5s
`set_security_level("delta")`
`sleep(25 SECONDS)` 25s
`addtimer(CALLBACK(GLOBAL_PROC, .proc/cult_ending_helper), 12 SECONDS)` 12s

this takes 77 seconds to trigger the actual cinematic

cinematic plays:

`flick("intro_cult",screen)`
`sleep(2.5 SECONDS)` 2.5s
`cinematic_sound(sound('sound/magic/enter_blood.ogg'))`
`sleep(2.8 SECONDS)` 2.8s
`cinematic_sound(sound('sound/machines/terminal_off.ogg'))`
`sleep(2 SECONDS)` 2.0s
`flick("station_corrupted",screen)`
`cinematic_sound(sound('sound/effects/ghost.ogg'))`
`sleep(7 SECONDS)` 7.0s
`special()`

this totals out to be 91.3s 
so at exactly 91.3 seconds after round ends, `special()` triggers

`special()` actually doesn't do anything anymore but it's a good signifier of when the cinematic probably ends if not a little sooner

it's fine

## Time Defines

Changed every instance of sleep() and animate() to use SECONDS and MINUTES where applicable

Changed some other things that caught my eye as I went through but didn't catch every instance

To-Do in future PRs:
- sleep( **DONE**
- animate( **DONE**
- SpinAnimation(
- spawn(
- flash_color(3rd arg
- clear_fullscreen(2nd arg
- addtimer(2nd arg
- QDEL_IN(2nd arg
- spin(both args
- SetRecoveryTime(both args
- SLEEP_CHECK_DEATH(
- Shake(

- duration var on temp effects

- Paralyze(
- Knockdown(
- Stun(
- other status procs

- many many other procs

# Changelog

:cl:  
tweak: Round now ends on Nar-sie summon for real this time
/:cl:

### CODE REVIEW

**Comment by Bibby0110** [comment]
Use time defines

**Comment by ynot01** [comment]
> Use time defines

i hate you

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -562,7 +562,7 @@
 		spawn(0)
```
Comment (tattax) [inline code/game/objects/items/toys.dm:562]:
```suggestion
		spawn(0 SECONDS)
```

**Review by tattax — COMMENTED** [review]
now do spawns

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (ynot01) [reply code/game/objects/items/toys.dm:562]: ill add it to the list

**Comment by Bibby0110** [comment]
atomize the time changes

**Comment by ynot01** [comment]
> atomize the time changes

they are atomized they're in different commits

**Review by Bibby0110 — CHANGES_REQUESTED** [review]
atomize the time commits into a separate PR :)

**Comment by ynot01** [comment]
no

**Comment by ynot01** [comment]
ok fine

**Comment by Bibby0110** [comment]
yay

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**